$OpenBSD: patch-build_config_rb,v 1.2 2017/11/01 21:44:44 jeremy Exp $

Don't build in debug mode by default.

Only build binaries we plan to use.

Index: build_config.rb
--- build_config.rb.orig
+++ build_config.rb
@@ -8,7 +8,7 @@ MRuby::Build.new do |conf|
     toolchain :gcc
   end
 
-  enable_debug
+  #enable_debug
 
   # Use mrbgems
   # conf.gem 'examples/mrbgems/ruby_extension_example'
@@ -107,7 +107,7 @@ MRuby::Build.new('host-debug') do |conf|
 
   # bintest
   # conf.enable_bintest
-end
+end if false
 
 MRuby::Build.new('test') do |conf|
   # Gets set by the VS command prompts.
@@ -134,7 +134,7 @@ MRuby::Build.new('bench') do |conf|
   end
 
   conf.gembox 'default'
-end
+end if false
 
 # Define cross build settings
 # MRuby::CrossBuild.new('32bit') do |conf|
